packages/typography/src/styled/StyledParagraph.ts

```diff
@@ -19,6 +19,7 @@ export const StyledParagraph = styled.p.attrs({
   margin: 0;
   padding: 0;
   direction: ${props => (props.theme.rtl ? 'rtl' : 'ltr')};
+  color: ${props => getColor({ theme: props.theme, variable: 'foreground.default' })};
```

Comment:
Sets default color correctly based on mode.

Reply:
ditto ..  paragraph `color` should be inherited